vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,31 @@ Upgrading Vonk
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Vonk.
 
+.. _vonk_releasenotes_390:
+
+Release 3.9.0
+-------------
+
+Features
+^^^^^^^^
+#. We have made :ref:`feature_subscription` more robust. If an evaluation of a Subscription fails, Vonk will retry the evaluation periodically for a number amount of tries.
```

Comment:
This is now configurable, right? If yes, I would mention the setting.

Reply:
Added a bit about the new Subscription settings